alert_test.go

```diff
@@ -147,6 +147,58 @@ func TestClient_CreateAlertRuleWithNotifications(t *testing.T) {
 	assert.Equal(t, &expected, res)
 }
 
+func TestClient_CreateAlertRuleWithWebhookNotifications(t *testing.T) {
+	setup()
+	out := `{"alertRuleId": 1, "ruleName": "test", "notifications":{"webhook":[{"integrationId": "1", "integrationName": "foo", "integrationType":"bar"},{"integrationId": "2", "integrationName": "foo2", "integrationType":"bar2"}]}}`
```

Comment:
Sorry to be picky but could you please use realistic values that illustrate how someone might use this Webhook instead of using "foo", "bar" etc?

Reply:
No worries, I've updated now.

Reply:
Thanks